### PR TITLE
Hades enhancements #1

### PR DESCRIPTION
**Pull request by szilard-nemeth**

### CODE REVIEW

File: cli.py [hunk 1]
```diff
@@ -184,6 +186,17 @@ def log(ctx, selector: str, follow: bool, tail: int or None, grep: str or None,
     handler.log(selector, follow, tail, grep, download)
 
 
+@cli.command()
```
Comment (9uapaw) [inline cli.py:199]: As this is yarn specific, this should be nested under the yarn subcommand, see queue for example:
```python
@yarn.command()
@click.pass_context
def queue(ctx):
```

File: cli.py [hunk 2]
```diff
@@ -228,12 +241,42 @@ def run_app(ctx, app: str, cmd: str = None, queue: str = None):
...
+        file_handler.formatter = None
+        logger.info("Logging to file: %s", file_handler.baseFilename)
+        handlers.append(file_handler)
+        logging.basicConfig(force=True, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=level, handlers=handlers)
+    else:
+        workdir = os.getcwd()
+    handler.run_script(script, workdir=workdir)
+
+
+@cli.command()
```
Comment (9uapaw) [inline cli.py]: Same here.

**Review by 9uapaw — COMMENTED** [review]
Thank you @szilard-nemeth for the huge contribution, appreciated. I had two minor comments before merging it.

File: cli.py (hunk 1 above)
Comment (szilard-nemeth) [reply cli.py:199]: fixed.

File: cli.py (hunk 2 above)
Comment (szilard-nemeth) [reply cli.py]: fixed.

**Pull request merged**